src/frontend/next/src/components/BannerDynamicItems.tsx

```diff
@@ -12,13 +12,13 @@ const useStyles = makeStyles((theme: Theme) => ({
   stats: {
     position: 'absolute',
     color: theme.palette.primary.contrastText,
-    fontFamily: 'Roboto',
-    fontSize: '2rem',
+    // fontFamily: 'Roboto',
```

Comment:
Are you using this? if not, I think it's better to remove it than just commenting it out

Reply:
I usually leave those commented code that way when everything isn't sure/done yet. I'll remove when all good